src/main/java/com/n1303/birthdaybuddy/controller/FriendsController.java

```diff
@@ -0,0 +1,28 @@
+package com.n1303.birthdaybuddy.controller;
+
+import com.n1303.birthdaybuddy.common.constant.UrlConstant;
+import com.n1303.birthdaybuddy.entity.Friendship;
+import com.n1303.birthdaybuddy.service.FriendService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(UrlConstant.FRIEND_API_BASE_URL)
+public class FriendsController {
+
+    @Autowired
+    private FriendService friendService;
+
+    @GetMapping(UrlConstant.ALL_USER_API_URL)
+    public List<Friendship> getAll() {
+        return this.friendService.getAll();
+    }
+
+    @PostMapping(UrlConstant.ADD_FRIENDSHIP)
+    public void insert(@RequestParam("firstUser") String firstUser,
```

Comment:
Would suggest returning a Friendship here wrapped in ResponseEntity object as done as done in TestController, same for above api.

Reply:
Sure